### PR TITLE
Add ExecuteStoreSql extension method to RelationalDatabaseFacade

### PR DESCRIPTION
**Pull request by mikary**
Adding new extension for #1863

### CODE REVIEW

File: src/EntityFramework.Relational/Infrastructure/RelationalStoreSqlExecutor.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            [NotNull] ISqlStatementExecutor statementExecutor,
+            [NotNull] IRelationalConnection connection,
+            [NotNull] IRelationalTypeMapper typeMapper)
+        {
+            _statementExecutor = statementExecutor;
+            _connection = connection;
+            _typeMapper = typeMapper;
+        }
+
+        public virtual void ExecuteStoreSql([NotNull] string sql, [NotNull] params object[] parameters)
```
Comment (divega) [inline src/EntityFramework.Relational/Infrastructure/RelationalStoreSqlExecutor.cs]: This is the equivalent to dbContext.Database.ExecuteSqlCommand in EF6. We may come up with a better name but I would suggest we keep it the same until we can discuss it in API review.

Comment (divega) [inline src/EntityFramework.Relational/Infrastructure/RelationalStoreSqlExecutor.cs]: (of course I was trying to say that about the actual method in RelationalDatabaseFacadeExtensions, but regardless, we should be consistent).

**Comment by mikary** [comment]
Could I have you take a look at this one @AndriySvyryd ?

**Comment by AndriySvyryd** [comment]
:shipit:

**Pull request merged**